### PR TITLE
Add SIG Scheduling API reviewers to OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by bsalamat**
**What type of PR is this?**
/kind cleanup

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/sig scheduling

### CODE REVIEW

**Comment by liggitt** [comment]
xref https://groups.google.com//forum/#!topic/kubernetes-sig-leads/xoMawT5WIgE

cc @k82cn 
cc @kubernetes/sig-scheduling-api-reviews 

/approve
per https://github.com/kubernetes/community/tree/master/sig-scheduling#leadership

will leave lgtm to @k82cn

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -372,8 +372,8 @@ aliases:
   #   - 
   
   # sig-scheduling-api-reviewers:
```
Comment (liggitt) [inline OWNERS_ALIASES]: uncomment this line as well

Comment (bsalamat) [reply OWNERS_ALIASES]: done

**Comment by k82cn** [comment]
/lgtm

My pleasure to contribute :)

**Comment by liggitt** [comment]
@bsalamat @k82cn - I'd suggest joining https://groups.google.com/forum/#!forum/kubernetes-api-reviewers as well, discussions of edge cases tend to happen there and are good to keep up with

**Comment by k82cn** [comment]
@liggitt , sure :)

**Pull request merged**